cmd/traffic/cmd/manager/internal/state/state.go

```diff
@@ -688,6 +688,9 @@ func (s *State) AgentTunnel(ctx context.Context, clientSessionInfo *rpc.SessionI
 		if !ok {
 			return nil, nil, status.Errorf(codes.NotFound, "client session %q not found", clientSessionID)
 		}
+		if cs.ClientTunnelServer == nil {
```

Comment:
@johnlanni can you please describe how you ended up with `cs.ClientTunnelServer` being `nil`  at this point? I can't see how that could happen. The `AgentTunnel` is created during an intercept and an intercept cannot be made unless the `ClientTunnelServer` is active.

Reply:
@thallgren I'm also confused about this, but seeing this line caused panic, the only possibility is that `ClientTunnelServer` is nil. The `cs` and `msg` variables cannot be nil pointers.
```go
 if err = cs.ClientTunnelServer.Send(msg.TunnelMessage()); err != nil {
```

Reply:
After I upgraded my telepresence to 2.3.6, this problem has not happened yet.